lib/truffleruby-tool/lib/truffle_tool.rb

```diff
@@ -461,7 +461,7 @@ def self.build_option_parser(parser_options, options_hash, option_parser: Option
       *args, description, block, default = data
 
       option_parser.on(*args, description + " (default: #{default.inspect})") do |new_value|
-        old_value            = options_hash[option]
+        old_value = options_hash[option]
```

Comment:
This is an example where I would wonder why is the `=` so far, and imho is less readable with the extra indentation.